src/site/content/en/installable/codelab-apple-touch-icon/index.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
How about we let @robdodson be the tiebreaker?

Here are my arguments for the codelab:

* It's useful to have a minimal, reproducible example of this behavior. I will probably need this in the future if I ever need to update this audit reference.
* It's good for SEO. This codelab will be the entry point for anyone wanting to add this functionality to their site. It will also give them the opportunity to discover Lighthouse.
  * So yes, from the Lighthouse UI > Codelab workflow, there's not much value. But from the Google Search > Codelab discovery workflow, I see value.
* The codelab clearly demonstrates the process for triggering the add to home screen workflow (**Share** > **Add to Home Screen**). This is not covered in the audit reference and I don't think it should be.